package/CHANGELOG

```diff
@@ -21,6 +21,8 @@ The rules for this file:
   * 2.0.0
 
 Fixes
+  * UserWarning/ValueError displayed when empty atomgroup is given 
+    to DensityAnalysis (Issue #3055)
```

Comment:
A `ValueError` is not displayed but raised. Make clearer when the warning is shown and when ValueError is raised.